### ISSUE TITLE
Need checking of QWe and APV sign

### ISSUE THREAD

**Issue by wdconinc** [JeffersonLab/remoll#86]
Email discussion with @seamusriordan caused me to realize we may need to check the sign of both the weak charges in remollPhysicalConstants.hh and the parity-violating asymmetry, and this for Moller, for elastic Al and for elastic p.

Moller:
![image](https://user-images.githubusercontent.com/4656391/40626408-63cbe8b0-6286-11e8-972d-21864a8cc89d.png)
Elastic ep
![image](https://user-images.githubusercontent.com/4656391/40626416-6f6af990-6286-11e8-99de-2b44775f259e.png)
Aluminum DS
![image](https://user-images.githubusercontent.com/4656391/40626422-7b72dce4-6286-11e8-9fef-95e6dcb98097.png)
Aluminum US
![image](https://user-images.githubusercontent.com/4656391/40626429-86ada288-6286-11e8-90fd-728a8f9f3e65.png)

**Comment by cameronc137** [comment]
@rahmans1 has noticed that the Moller Asymmetry has switched from a positive sign to a negative sign at some point in the last year. Has there been an update on this issue recently?

**Comment by wdconinc** [comment]
E158 (https://arxiv.org/abs/hep-ex/0504049) measured a negative asymmetry of -131 +/- 14 (stat.) +/- 10 (syst.) parts per billion by using the definition:
![image](https://user-images.githubusercontent.com/4656391/43675635-3af8eeee-97db-11e8-8f44-d9414994fcde.png)

MOLLER (http://hallaweb.jlab.org/12GeV/Moller/moller_final.pdf) uses:
![image](https://user-images.githubusercontent.com/4656391/43675655-9cf9a7d2-97db-11e8-97c2-57eb69256ba0.png)

Apart for positive kinematic factors, 2 m E y = Q^2, it's clear that there's a sign difference. In my opinion, the sign of Q_W^e used in the MOLLER proposal is wrong, and should be (4 sin^2 theta_W - 1) instead. That is what we use in include/remollPhysicalConstants.hh
```
const double QWe =  -0.0435;  // arxiv 1302.6263 - KK's low Q2 EW review
```
and which is turned into an asymmetry in src/remollGenMoller.ss
```
G4double APV = electron_mass_c2*beamE*GF*4.0*sin(thcom)*sin(thcom)*(QWe+QWe_rad)/(sqrt(2.0)*pi*alpha*pow(3.0+cos(thcom)*cos(thcom),2.0));
```

In summary:
- Moller ee scattering: correct sign
- Elastic ep scattering: correct sign
- Aluminum scattering: correct sign (but factor 10 larger than 1.9 ppm observed in Qweak)

**Comment by wdconinc** [comment]
For aluminum elastic scattering:
```
  asym= -gf/(4.*pi*fine_structure_const*sqrt(2.)) * Q2/GeV/GeV * (qwp+qwn*(A-Z)/Z);
```
but where now qwn dominates and is essentially -1. A positive asymmetry ensues. The change in Q^2 between Qweak (APV(eAl) = 1.9 ppm) and here is about 11^2 due to E_in*E_out but similarly there's a factor (12 deg / 20 mrad)^2 due to the angles that cancels this...

**Comment by cameronc137** [comment]
Ok, I think I remember following some discussion of this in meetings a while ago. Has this change of sign been incorporated into remoll? And if so, for what release version (or commit if it is older) does it get introduced?

**Comment by wdconinc** [comment]
It got introduced a looooong time ago (2014 if I recall). But because some branches diverged from master even loooooooonger ago this issue just keeps cropping up when people use those branches. It has been correct in every release.

**Comment by wdconinc** [comment]
Qwe originally introduced with incorrect sign in aaacad6a6df8016d6c922d4b9c8f2de96eced0dd

Sign fixed in aaacad6a6df8016d6c922d4b9c8f2de96eced0dd
